qos/evm/response_generic.go

```diff
@@ -0,0 +1,46 @@
+package evm
+
+import (
+	"encoding/json"
+
+	"github.com/buildwithgrove/path/qos/jsonrpc"
+)
+
+// responseUnmarshallerGeneric unmarshal the provided byte slice
+// into a responseGeneric struct and saves any data that may be
+// needed for producing a response payload into the struct.
+func responseUnmarshallerGeneric(data []byte) (response, error) {
```

Comment:
ULTRA NIT: I think you shold put this function at the bottom. I want to start reading w/ an understanding of what this file is about.